libcnb/src/lib.rs

```diff
@@ -78,6 +82,39 @@ macro_rules! buildpack_main {
     };
 }
 
+/// Resolves the path to an additional buildpack binary by Cargo target name when the buildpack is
```

Comment:
This sentence is quite hard for me to read. I think some additional punctuation or moving the `libcnb-cargo` / `libcnb-test` out of the opening line, might help?

Also, would it be worth showing an example usage here? I presume it would need to be `no_run` (since it wouldn't compile with a made up binary name), but it might at least make the purpose clearer? :-)

Reply:
What about `compile_fail`? (TIL)
https://doc.rust-lang.org/rustdoc/documentation-tests.html?#attributes

Reply:
I added an example with an explanation why it is marked as failing. (There is a big red exclamation mark next to the example in the docs when `compile_fail` is used).

Reply:
I removed the libcnb-cargo info from the opening line. The libcnb-cargo/test caveat is already a note in the body and IMO enough. :)